Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamer.cpp

```diff
@@ -735,8 +735,7 @@ void webKitMediaSrcPrepareSeek(WebKitMediaSrc* source, const MediaTime& time)
 
     for (Stream* stream : source->priv->streams) {
         stream->appsrcNeedDataFlag = false;
-        // Don't allow samples away from the seekTime to be enqueued.
-        stream->lastEnqueuedTime = time;
+        stream->lastEnqueuedTime = MediaTime::invalidTime();
```

Comment:
Hi @eocanha, thanks for reviewing this.

lastEnqueuedTime is not used anywhere to check the enqueuing sample(maybe it was used before?). So this change doesn't break anything. the purpose is to avoid the flush when there were no samples enqueued. 

As for webKitMediaSrcPrepareInitialSeek, it doesn't actually result in a flushing seek. so I left it unchanged. I think it makes sense to just remove the initializatoin of lastEnqueuedTime from webKitMediaSrcPrepareInitialSeek. let me update the commit.

Reply:
updated webKitMediaSrcPrepareInitialSeek